### PR TITLE
Fix Bug on Exclude

### PR DESCRIPTION
**Pull request by leandromoh**
Passing parameter `count` with value zero should return the original sequence, but it doesnt. 

`Enumerable.Range(0, 6).Exclude(3, 0)` returns `[0, 1, 2, 4, 5]` instead of `[0, 1, 2, 3, 4, 5]`

### CODE REVIEW

File: MoreLinq.Test/ExcludeTest.cs [hunk 1]
```diff
@@ -39,6 +39,18 @@ public void TestExcludeNegativeCountException()
...
         /// <summary>
+        /// Verify that excluding with count equals zero returns the original source
+        /// </summary>
+        [Test]
+        public void TestExcludeWithCountEqualsZero()
+        {
+            var sequence = Enumerable.Range(1, 10);
+            var resultA = sequence.Exclude(5, 0);
+
+            Assert.IsTrue(resultA.SequenceEqual(sequence));
```
Comment (atifaziz) [inline MoreLinq.Test/ExcludeTest.cs]: A more accurate test would be:

```c#
Assert.That(resultA, Is.SameAs(sequence));
```

**Comment by atifaziz** [comment]
> b8c45c0a7f89e9eb7154a8f933b0144ee469bdbe should have been a separate PR as it is unrelated to the bug. Could you please revert it from here? I'll cherry-pick it on to master.

Never mind. It looks like all changes from b8c45c0a7f89e9eb7154a8f933b0144ee469bdbe are already in master as 474d72ec4db6482a3a4e6cf9d1edf42f2d32d904.

**Comment by leandromoh** [comment]
@Atifaziz When do you plan to merge? 
I AM waiting this to submit the PR of SwapRange

**Comment by atifaziz** [comment]
@leandromoh As soon as the [small change requested to the test](https://github.com/morelinq/MoreLINQ/pull/321#discussion_r122634202) is done.

**Comment by leandromoh** [comment]
done! sorry, I did not understand that that was a request, just a comment.

**Comment by atifaziz** [comment]
Thanks!

**Pull request merged**